### ISSUE TITLE
ES Crashes

### ISSUE THREAD

**Issue by lilbud** [jacobfk20/EmulationStation-RPiE#4]
Your latest build of ES crashed and gave me this error
[http://i.imgur.com/lVS4L3e.png](http://i.imgur.com/lVS4L3e.png)

**Comment by jacobfk20** [comment]
What were you doing?  Was this on boot?  No controllers plugged in?
Also, this is my older fork of ES.  Are you using this version? (with old gridview, OSK, WifiConnect?)

**Comment by jacobfk20** [comment]
Nevermind on what fork..  I saw you're running ESGbuild.  That branch doesn't touch InputManager..  But still give me some more info on what exactly you were doing.

**Comment by lilbud** [comment]
I am using the SFC 30 bluetooth controller. Upon connecting, ES will crash out to the command line.

**Comment by jacobfk20** [comment]
It doesn't do this in the main retropie fork of ES?  
Is this a new controller?

**Comment by lilbud** [comment]
No, have been using it for about a couple months. I updated the setup-script today. Do you think that has something to do with it

**Comment by jacobfk20** [comment]
I wouldn't think so.  Though I don't really know what goes on in those updates.  I'll take a look at the update logs and see if it could possibly be related.  
Are you using a Pi3 with its internal bluetooth?

**Comment by lilbud** [comment]
No, using external dongle.

**Comment by jacobfk20** [comment]
Okay, One more thing, I know I already asked but I want to be sure, It doesn't crash when running the original ES After you've updated the setup-script, correct?

**Comment by lilbud** [comment]
No it doesn't. Never had this issue before

**Comment by jacobfk20** [comment]
Will it crash if the controller is already connected before emulationstation starts?

**Comment by lilbud** [comment]
The controller will not connect before ES starts. At least not for me.

**Comment by jacobfk20** [comment]
I'm going to update my setup-script and see if it does it for me.  I'm also using bluetooth.  I may send you a special version of ES that will report exactly what is going wrong there.

**Comment by jacobfk20** [comment]
I updated to the newest setup script and upgraded the kernel and it still runs fine for me..  I'll send you a special debug build tomorrow that will spit out more useful information for me to track this problem down.

**Comment by jacobfk20** [comment]
I posted a debug version that will give a little bit more information to my drive.
https://drive.google.com/open?id=0ByWG5kDXyr5NWWFTT2Y0dDdTdms
